scripts/hid_configurator/gui/gui.py

```diff
@@ -128,11 +140,14 @@ def dfu(self, update_progressbar):
         print(self.filepath)
         success = self.device.perform_dfu(self.filepath, update_progressbar)
         if success:
-            print('DFU transfer completed')
-            dfu_label.text = 'DFU transfer completed'
+            info = 'DFU transfer completed'
+            message = 'You can now go back to application and switch firmware'
```

Comment:
Maybe `Firmware will be switched on next reboot`?

Reply:
ok, so I also change the name of the button from 'Switch to new firmware' to 'Firmware reboot'